app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -86,6 +88,11 @@ class PocketVideoFragment : Fragment() {
         }
     }
 
+    override fun onStop() {
+        super.onStop()
+        job?.cancel()
```

Comment:
I don't think this will work: I think this job either:
- Needs to be restarted in onStart if it does not complete
- Needs to be destroyed in onDestroyView instead

The former is probably more correct but more complex.

Reply:
Hm.  We could move the existing setup logic from `onCreateView` into `onStart`.  That does feel more correct, wouldn't be a big task, and shouldn't impact functionality (we're creating a new instance every time anyway).  Are you ok with that solution?